README.md

```diff
@@ -82,6 +85,20 @@ fastify.register(require('fastify-rate-limit'), {
 })
 ```
 
+`errorMessage` example usage:
+```js
+fastify.register(require('fastify-rate-limit'), {
+  /* ... */
+  errorMessage: function(after, max) {
```

Comment:
I would directly add `req` as a first parameter, or change the approach a single object, such as `context`:
```js
{
  after: String,
  max: Number
}
```
The second approach is more future proof :)

Reply:
`req` does not contain the `after` or `max`, but might be also needed/nice to have for building error messages.
Is it then better to have to parameter `(req, context)` or shall `req` be part of `context`?
```ts
{
  req: fastify.FastifyRequest<http.IncomingMessage>,
  after: string,
  max: number
}
```

Reply:
@delvedor ping :-)